package.json

```diff
@@ -4,7 +4,8 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "start": "nodemon --exec babel-node src/index.js",
+    "start": "nodemon --exec babel-node src/server.js",
+    "dev": "nodemon --exec babel-node src/server.js",
```

Comment:
We should run the index.js file and not server file.

Reply:
Okay. The code that starts the server has been moved to the index.js file and server.js has been deleted